convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -146,14 +146,21 @@ def run(self):
         # shows which packages were not removed, if false, all packages were removed
         pkgs_not_removed = sorted(frozenset(pkghandler.get_pkg_nevras(all_pkgs)).difference(pkgs_removed))
         if pkgs_not_removed:
-            message = "The following packages were not removed: %s" % ", ".join(pkgs_not_removed)
+            message = "The following packages cannot be removed: %s" % ", ".join(pkgs_not_removed)
             logger.warning(message)
             self.add_message(
                 level="WARNING",
                 id="SPECIAL_PACKAGES_NOT_REMOVED",
-                title="Special packages not removed",
-                description="Special packages which could not be removed",
-                diagnosis=message,
+                title="Some packages cannot be removed",
+                description=message,
+                diagnosis=(
+                    "The packages match a pre-defined list of packages that are to be removed during the"
+                    " conversion. This list includes packages that are known to cause a conversion failure."
+                ),
+                remediations=(
+                    "We do a conversion dry-run (yum transaction validation) later on as part of the pre-conversion"
+                    " analysis. If the dry-run fails, try removing the packages listed above manually."
```

Comment:
Do we want to add there also command customer can use?

Reply:
The first sentence in the remediations seems to be they would be a better fit in the diagnosis, I belive... Right now, it doesn't seem that they are fitting in the remediations correctly. 

I tend to think that remediation would be a step-to-step on how to solve the problem, not really an explanation of why it failed.

> Do we want to add there also command customer can use?

If we know what command can be used, I don't see why not. Would be useful